zero/meta-csp/recipes-cspd/csp-handler/files/handler.c

```diff
@@ -43,6 +43,10 @@ void *handle_csp_packet(void *param)
 				csp_buffer_free(packet);
 				break;
 
+			case PORT_J:
+				capture_jpeg_service(conn);
+				csp_buffer_free(packet);
```

Comment:
Not for this PR but we must move csp_buffer_free() out of each case?

Reply:
I agree it.
I think it would be better to free it within `xx_service`, similar to the standard `csp_service_handler` in CSP.